lib/webdrivers/chromedriver.rb

```diff
@@ -115,14 +114,14 @@ def chrome_on_windows
         end
 
         # Default to Google Chrome
-        reg        = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe'
+        reg = 'HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe'
         executable = system_call("powershell (Get-ItemProperty '#{reg}' -Name '(default)').'(default)'").strip
         Webdrivers.logger.debug "Browser executable: '#{executable}'"
         ps = "(Get-Item (Get-ItemProperty '#{reg}').'(default)').VersionInfo.ProductVersion"
         system_call("powershell #{ps}").strip
```

Comment:
I think we can remove `strip` from these methods to avoid the error message reported in #67. Line 33 will strip the spaces anyway.

Reply:
same on line 133 and 139.

Reply:
Since that line isn't part of this PR, let's resolve this for now and do a sweep for any unnecessary commands before we release 3.9.

Reply:
Sounds good!